getting-started/windows-calico/cluster-roles.md

```diff
@@ -0,0 +1,84 @@
+---
+title: Create cluster roles for Windows nodes
+description: Configure cluster roles with correct permissions for Calico for Windows.
+canonical_url: '/getting-started/windows-calico/cluster-roles'
+---
+
+### Big picture
+
+Create cluster role for Windows nodes.
+
+### How to
+
+Because Kubernetes on Windows cannot run {{site.prodname}} in a pod with an auto-provisioned service account, {{site.prodname}} requires a kubeconfig file to access the API server. This section describes how to create an appropriate service account, and then to export the service account token as a kubeconfig file for {{site.prodname}} to use.
+
+>**Note**: In general, the node kubeconfig as used by kubelet does not have enough permissions to access {{site.prodname}}-specific resources.
+{: .alert .alert-info}
+
+#### Install the cluster-role manifest
+
+A cluster-role with the correct permissions for {{site.prodnameWindows}} is available here: [calico-windows cluster role](https://github.com/projectcalico/calico/releases/download/v3.16.0/win-cluster-role.yaml).
+
+Apply the cluster role.
+
+```
+$ kubectl apply -f
```

Comment:
@song-jiang do we still need a separate cluster role?

Reply:
Maybe we just need a cluster rolebinding?

Reply:
We should use existing `calico-node` secret as Calico for Windows has to work with Calico on Linux side. I will update the doc.